client/appengine_data.go

```diff
@@ -321,24 +324,115 @@ func (d *DatastreamPaginator) parseData(rawData []byte) any {
 }
 
 func parseDatastream(rawData []byte, aggregation interfaces.AstarteInterfaceAggregation) any {
-	data := gjson.ParseBytes(rawData).Array()
```

Comment:
Since `rawData` is parsed only once in `parseDatastreamWithIndividualAggregation` and `parseDatastreamWithObjectAggregation`, move back here the parsing of `rawData`

Reply:
Updated